controllers/process.go

```diff
@@ -180,7 +185,7 @@ func (r *ServicefenceReconciler) ReconcileNamespace(req ctrl.Request) (ret ctrl.
 		nsLabel = ns.Labels[LabelServiceFenced]
 	}
 
-	nsFenced := nsLabel == ServiceFencedTrue
+	nsFenced := nsLabel != ServiceFencedFalse
```

Comment:
we should decouple the explicit configuration at the namespace/service level from the role of global configuration, i.e. decouple the code and combine it at the upper level sometime later